Formula/storj-uplink-cli.rb

```diff
@@ -0,0 +1,21 @@
+class StorjUplinkCli < Formula
+  desc "Client-side application that supports CLI interaction with the Storj DCS network"
+  homepage "https://github.com/storj/storj/wiki/Uplink-CLI"
+  url "https://github.com/storj/storj/releases/latest/download/uplink_darwin_amd64.zip"
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
My apologies for not responding right away, and sorry about the binary thing I misinterpreted the documentation, thinking as an open source cross platform Go app that it fit. 

Would it be best I move it to homebrew-cask or keep it in a tap outside core?

sorry for the extra work. I’ve made a couple  casks but this is my first core formula, and hoped I would have did a bit better. At least learned something though!

Reply:
> Would it be best I move it to homebrew-cask or keep it in a tap outside core?

If it's open source you can just compile it no?